client/src/components/Auth/Login.tsx

```diff
@@ -20,9 +20,7 @@ function Login() {
     }
   }, [isAuthenticated, navigate]);
 
-  const SERVER_URL = import.meta.env.DEV
-    ? import.meta.env.VITE_SERVER_URL_DEV
-    : import.meta.env.VITE_SERVER_URL_PROD;
+  const SERVER_URL = import.meta.env.DOMAIN_SERVER;
```

Comment:
this has to be prefaced with VITE_ , otherwise it will not be read

Reply:
@danorlando I changed it so that you can pass through DOMAIN_*, SCRIPT_*, and VITE_* vars as they are safe prefixs and reduces the duplicate env vars while removing the vite dependancy

Reply:
Oh cool. yea, i see that now.